### ISSUE TITLE
snipmate is working on GUI but not on terminal

### ISSUE THREAD

**Issue by tiagonbotelho** [garbas/vim-snipmate#193]
When i load the snipmate on gui it works perfectly but when i load on my terminal it does not find the snippets.

I get this error when i run SnipMateOpenSnippetFiles:
```
01# /Users/tiagobotelho/.vim/bundle/vim-snippets/snippets/_.snippets
  2 02# does not exist: /Users/tiagobotelho/.vim/snippets/_.snippets
  3 03# does not exist: /Users/tiagobotelho/.vim/bundle/Vundle.vim/snippets/_.snippets
  4 04# does not exist: /Users/tiagobotelho/.vim/bundle/nerdtree/snippets/_.snippets
  5 05# does not exist: /Users/tiagobotelho/.vim/bundle/vim-addon-mw-utils/snippets/_.snippets
  6 06# does not exist: /Users/tiagobotelho/.vim/bundle/tlib_vim/snippets/_.snippets
  7 07# does not exist: /Users/tiagobotelho/.vim/bundle/vim-snipmate/snippets/_.snippets
  8 08# does not exist: /Users/tiagobotelho/.vim/bundle/ctrlp.vim/snippets/_.snippets
  9 09# does not exist: /Users/tiagobotelho/.vim/bundle/auto-pairs/snippets/_.snippets
 10 10# does not exist: /Users/tiagobotelho/.vim/bundle/powerline/snippets/_.snippets
 11 11# does not exist: /Users/tiagobotelho/.vim/bundle/vim-hackernews/snippets/_.snippets
 12 12# does not exist: /usr/share/vim/vimfiles/snippets/_.snippets
 13 13# does not exist: /usr/share/vim/vim73/snippets/_.snippets
 14 14# does not exist: /usr/share/vim/vimfiles/after/snippets/_.snippets
 15 15# does not exist: /Users/tiagobotelho/.vim/after/snippets/_.snippets
 16 16# does not exist: /Users/tiagobotelho/.vim/bundle/Vundle.vim/snippets/_.snippets
 17 17# does not exist: /Users/tiagobotelho/.vim/bundle/Vundle.vim/after/snippets/_.snippets
 18 18# does not exist: /Users/tiagobotelho/.vim/bundle/nerdtree/after/snippets/_.snippets
 19 19# does not exist: /Users/tiagobotelho/.vim/bundle/vim-addon-mw-utils/after/snippets/_.snippets
 20 20# does not exist: /Users/tiagobotelho/.vim/bundle/tlib_vim/after/snippets/_.snippets
 21 21# does not exist: /Users/tiagobotelho/.vim/bundle/vim-snipmate/after/snippets/_.snippets
 22 22# does not exist: /Users/tiagobotelho/.vim/bundle/vim-snippets/after/snippets/_.snippets
 23 23# does not exist: /Users/tiagobotelho/.vim/bundle/ctrlp.vim/after/snippets/_.snippets
 24 24# does not exist: /Users/tiagobotelho/.vim/bundle/auto-pairs/after/snippets/_.snippets
 25 25# does not exist: /Users/tiagobotelho/.vim/bundle/powerline/after/snippets/_.snippets
 26 26* does not exist: /Users/tiagobotelho/.vim/bundle/vim-hackernews/after/snippets/_.snippets
```

**Comment by ajzafar** [comment]
@tiagonbotelho That is not an error. That's the output of the `:SnipMateOpenSnippetFiles` command. It shows various possible locations for snippets files then asks you to pick one to start editing.

Please describe *how* SnipMate isn't working. Just stating that it doesn't work doesn't really help fix the problem.

**Comment by tiagonbotelho** [comment]
well all i can say is that for istance when i type main(wich should make a main type function) in C, on the terminal it does not load the snippet when i press Tab, now when i do it on the GUI it happens to be th other way around, when i press Tab it does load it

**Comment by ajzafar** [comment]
@tiagonbotelho Could you run through the check list that I just added to the FAQ in the README? Nothing in SnipMate targets or references GVim in any way, so running through the checklist will help narrow down what differs between your terminal Vim and your GVim.

**Comment by tiagonbotelho** [comment]
I ran through it but i had little success it does in fact print the :imap correctly but when i try to use a snippet it still wont load :/
I have here my .vimrc file if you wish to see if there is something wrong with it but my directories are correct imo, atleast Vundle is installing them in the right place :)

https://github.com/tiagonbotelho/_vimrc

**Comment by ajzafar** [comment]
@tiagonbotelho It looks like you're using UltiSnips (now)? I think that's a fine choice, but then this issue should be closed.

If you're still using SnipMate, please try everything in the FAQ. In particular make sure your snippets are where they should be and check if you can see *any* snippets.

**Comment by tiagonbotelho** [comment]
It worked i just had the wrong vim version, it works now fully it was not looking for vim on my brew installation thanks for the help!

**Comment by elig0n** [comment]
I have the same issue with
`VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Feb 24 2022 18:41:57)`
`g:plugs['vim-snipmate'].commit = 'ed3c5426a20bf1c06d7980946ada34fd9f93320e'`
installed with plug:
```
Plug 'garbas/vim-snipmate'
Plug 'honza/vim-snippets'
```

I went through all the FAQ's checklist with no special problems.
All mappings I tried to set for `<Plug>snipMateNextOrTrigger` and `C-R Tab` works only in gvim.
The mappings are seen in `:imap` and work for other stuff. The current filetype snippets are correctly shown with `:SnipMateOpenSnippetFiles`
